environment.yml

```diff
@@ -5,6 +5,7 @@ channels:
 - astropy
 dependencies:
 - python=3.6
+- hitran-api
```

Comment:
```suggestion
```

(can't modify this directly, it's out of scope, sorry !)

hitran-api is not a Conda package so you have to install it under the pip: section below (l27).

Reply:
Test failed with 
```
ResolvePackageNotFound: 

  - hitran-api
```
so that may be the reason !

Reply:
Aha gotcha!

Reply:
You still have to add it in the environment.yml, under the pip section, i.e. 

```

- pip
- pip:
  - publib>=0.3.2  # Plotting styles for Matplotlib
  - hitran-api  👈
```